dtmi/sgs/ocm_online-2.json

```diff
@@ -0,0 +1,275 @@
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "Fe700T800",
+      "displayName": "Fe700~800um",
+      "description": "700~800um Ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "Fe800T",
+      "displayName": "Fe>800um",
+      "description": ">800um Ferromagnetic particle count",
+      "schema": "integer"
+    },
+
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFe150T200",
+      "displayName": "NonFe 150~200um",
+      "description": "150~200um Non ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFe200T300",
+      "displayName": "NonFe 200~300um",
+      "description": "200~300um Non ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFe300T400",
+      "displayName": "NonFe 300~400um",
+      "description": "300~400um Non ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFe400T500",
+      "displayName": "NonFe 400~500um",
+      "description": "400~500um Non ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFe500T600",
+      "displayName": "NonFe 500~600um",
+      "description": "500~600um Non ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFe600T700",
+      "displayName": "NonFe 600~700um",
+      "description": "600~700um Non ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFe700T800",
+      "displayName": "NonFe 700~800um",
+      "description": "700~800um Non ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFe800T",
+      "displayName": "NonFe >800um",
+      "description": ">800um Ferromagnetic particle count",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "FeTotal24",
+      "displayName": "FeTotal24",
+      "description": "Total number of Ferromagnetic particles in 24 hours",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Count"
+      ],
+      "name": "NonFeTotal24",
+      "displayName": "NonFeTotal24",
+      "description": "Total number of Non ferromagnetic particles in 24 hours",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Flow"
```

Comment:
Although DTDL v2 does not have an appropriate semantic type, DTDL v3 uses "VolumeFlowRate" for this.  For forward-compatibility, VolumeFlowRate would be preferable to Flow.